R/pkg/R/SQLContext.R

```diff
@@ -602,23 +614,17 @@ loadDF <- function(path = NULL, source = NULL, schema = NULL, ...) {
 #' Create a SparkDataFrame representing the database table accessible via JDBC URL
 #'
 #' Additional JDBC database connection properties can be set (...)
+#' You can find the JDBC-specific option and parameter documentation for reading tables via JDBC in
+#' \href{https://spark.apache.org/docs/latest/sql-data-sources-jdbc.html#data-source-option}{
+#' Data Source Option} in the version you use.
 #'
 #' Only one of partitionColumn or predicates should be set. Partitions of the table will be
 #' retrieved in parallel based on the \code{numPartitions} or by the predicates.
 #'
 #' Don't create too many partitions in parallel on a large cluster; otherwise Spark might crash
 #' your external database systems.
 #'
-#' @param url JDBC database url of the form \code{jdbc:subprotocol:subname}
 #' @param tableName the name of the table in the external database
-#' @param partitionColumn the name of a column of numeric, date, or timestamp type
-#'                        that will be used for partitioning.
-#' @param lowerBound the minimum value of \code{partitionColumn} used to decide partition stride
-#' @param upperBound the maximum value of \code{partitionColumn} used to decide partition stride
-#' @param numPartitions the number of partitions, This, along with \code{lowerBound} (inclusive),
-#'                      \code{upperBound} (exclusive), form partition strides for generated WHERE
-#'                      clause expressions used to split the column \code{partitionColumn} evenly.
-#'                      This defaults to SparkContext.defaultParallelism when unset.
```

Comment:
Seems like CRAN forces to keep the documentation for parameters:

```
Undocumented arguments in documentation object 'read.jdbc'
  ‘url’ ‘partitionColumn’ ‘lowerBound’ ‘upperBound’ ‘numPartitions’

Undocumented arguments in documentation object 'write.jdbc'
  ‘url’
```

I think you can just keep these documentation for now.